brave/brave-core
# Open youtube in brave android

Resolves https://github.com/brave/brave-browser/issues/41958



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
1. Install a version with out the change in the PR
2. disable `Open YouTube links in Brave` option from Settings -> Media
3. Install a new version with the PR change
4. Observe the dialog on the first app open after an upgrade. 
5. Tap `Turn on now`, and observer the redirection to `youtube` page

Adding a recording as the dialog includes an animation. 
https://github.com/user-attachments/assets/60fef9a8-08ba-4bc7-b275-fc0dbec3a243
